### PR TITLE
DOC-1381: Onboarding Parking Services

### PR DESCRIPTION
**Pull request by martapederiva**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/DOC-1381

### *What is the problem we're trying to solve*

Adding the necessary configuration to onboard Parking Services to use DES

### *What changes have we introduced*

Config JSON files

### CODE REVIEW

File: auth-groups.json [hunk 1]
```diff
@@ -20,6 +21,7 @@
     "Admission and Pupil Benefits": "education-evidence-production",
     "Leasehold and Right to Buy": "Leasehold and Right to Buy",
     "Housing Management": "DES Housing Management",
+    "Hackney Parking Services": "Hackney Parking Services - Evidence Drop Box",
```
Comment (bogdan-zaharia) [inline auth-groups.json:24]: We shouldn't give them access to production yet. We will do that once they confirmed things are ok in staging.

**Review by bogdan-zaharia — COMMENTED** [review]
Left a minor comment

File: auth-groups.json (hunk 1 above)
Comment (martapederiva) [reply auth-groups.json]: Addressed in [aae11f0](https://github.com/LBHackney-IT/document-evidence-store-frontend/pull/293/commits/aae11f03020c1dceeec7f7cf1965932a3644521d)

**Review by bogdan-zaharia — APPROVED** [review]
LGTM

**Pull request merged**